careers/faq.md

```diff
@@ -5,24 +5,40 @@ sidebar_label: FAQ
 
-Co-op advisors advise against applying without their permission, but in the end there's nothing stopping you
-from applying for internships on your own. If you find one by yourself and want to stay in the co-op program,
-co-op will gladly accept that company as a co-op placement after a little screening.
+Up until now, the definition of Co-op and internship works best for Undergraduate and Master of Science students. However, Master of Applied Computing (MAC) students follow slightly different rules. Technically, MAC students are told they apply to internships only, but in reality, they can apply to both Co-op and internship roles as they are registered in a Co-op program. The reason that MAC students apply to "internships" relies on the second difference of Co-ops and internships; length of duration. Co-ops usually require 8 - 16 months of experience to graduate, though internships only require 4 months of experience. Undergradaute students require 3 work terms to graduate, and Master of Scieence students require 2 works term to graduate. However, MAC students only require 1 work term to graduate, hence why the Co-op office informs MAC students that they are applying for internships as the work term requirements are much shorter comapred to other programs.
+
+#### Should I pay for Co-op or just go for internships?
+
+Do both! Co-op is an excellent way to land your first Co-op/internship if you are jsut entering the world of tech. This is because there are jobs exclusive to students; therefore, those job postigs have smaller pools of competition and the expectations of the job are more junior in nature. You will also have a dedicated team of coordinators working in partnership with you to find and secure these opportunities where employers are provided with subsidies to hire students (i.e. they get a tax credit when hiring Co-op students). If you complete all 3 required Co-op work terms, you will will graduate with at least 12 months of full-time, relevant work experience giving you an edge in the job market post-graduation. If at any point in the program you decide to withdraw from Co-op, remember that you can no longer gain access to positions eligible for registered Co-op students, so keep this in mind when reviewing other opportunities outside of Co-op.
```

Comment:
```suggestion
Do both! Co-op is an excellent way to land your first Co-op/internship if you are just entering the world of tech. This is because there are jobs exclusive to students; therefore, those job postings have smaller pools of competition and the expectations of the job are more junior in nature. You will also have a dedicated team of coordinators working in partnership with you to find and secure these opportunities where employers are provided with subsidies to hire students (i.e. they get a tax credit when hiring Co-op students). If you complete all 3 required Co-op work terms, you will will graduate with at least 12 months of full-time, relevant work experience giving you an edge in the job market post-graduation. If at any point in the program you decide to withdraw from Co-op, remember that you can no longer gain access to positions eligible for registered Co-op students, so keep this in mind when reviewing other opportunities outside of Co-op.
```